### ISSUE TITLE
Add link to panvala gitcoin grant

### ISSUE THREAD

**Issue by GriffGreen** [commons-stack/conviction-voting-pilot#5]
Grant link might be wrong... Confirm that all links to the gitcoin grant are correct:
It should be: 
https://gitcoin.co/grants/1127/commons-stack-community-fund-panvala-league

The current link probably looks very similar but is 1128 (instead of 1127) and ends in a `-2`

I couldn't find the old link anywhere... but if it's not there somewhere... it should be :-D

**Comment by sembrestels** [comment]
Fortunately there is no reference to the panvala gitcoin grant in any of the conviction voting pilot code, and the link seems updaded in the [blog post](https://medium.com/commonsstack/cstk-token-holders-can-now-use-conviction-voting-on-xdai-5090fdc3b8ad). Is there somewhere else we can update the link?

**Comment by GriffGreen** [comment]
Yeah that's the question... in the end, the funding from this grant is what fuels the CV pilot... so we should probably find a place for it to be linked prominently...
https://gitcoin.co/grants/1127/commons-stack-community-fund-panvala-league
